_includes/parse-server/usage.md

```diff
@@ -18,7 +18,7 @@ var api = new ParseServer({
 
 The parameters are as follows:
 
-* `databaseURI`: Connection string URI for your MongoDB.
+* `databaseURI`: Connection string URI for your database.
```

Comment:
I don't think this is proper wording, "connection string" is just another word for "URI".

> * `databaseURI`: Connection string for your database.

Reply:
I never heard about 'Connection string' in my life 🤣, I just changed MongoDB for database..  in this file, I am gonna  change `var` for `const` https://stackoverflow.com/questions/44212934/what-is-the-best-way-to-require-node-modules-var-or-const